app/styles/components/_inplace-text.scss

```diff
@@ -0,0 +1,14 @@
+.inplace-text {
```

Comment:
This should be in _editinplace.scss since it is common styling for all inplace elements.